package.json

```diff
@@ -45,6 +45,7 @@
     "@rollup/plugin-json": "^4.0.3",
     "@rollup/plugin-node-resolve": "^8.0.0",
     "@rollup/plugin-replace": "^2.3.2",
+    "@types/jest": "^25.2.3",
```

Comment:
Ah, good idea!

Reply:
I had removed it, since `jest` now has its own types, but seems like that caused #145 😓